doc/integrator/authentication.rst

```diff
@@ -158,23 +156,23 @@ Some service of GeoMapFish has some host restriction if you mix the domain.
 Application authentication
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To be considered as authenticated we should have the correct `Cookie` header,
-we also check the `Referrer` header to be sure that the user is coming from the same domain.
-If he is equals to the `Host` header, we consider that the user is coming from the same domain.
+To be considered as authenticated we should have the correct ``Cookie`` header,
+we also check the ``Referrer`` header to be sure that the user is coming from the same domain.
```

Comment:
```suggestion
we also check the ``Referer`` header to be sure that the user is coming from the same domain.
```

Reply:
if this change is the result of automatic orthographic correction, cant we add a global rule to ignore that particular word ?

Reply:
+1 :-)